examples/tutorials/flow-as-a-function/sample.ipynb

```diff
@@ -78,6 +78,50 @@
     "\n",
     "print(result)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from promptflow import load_flow\n",
+    "from promptflow.entities import Flow\n",
+    "\n",
+    "# load the flow\n",
+    "f: Flow = load_flow(\n",
+    "   source='xx.yaml', \n",
+    "   # user can specify variant here to load a specific variant\n",
+    "   variant=\"${summarize_text_content.variant_0}\",\n",
```

Comment:
I think we don't support variant here?

Reply:
Good catch! Forgot to update here

Reply:
Seems some of my local changes lost, updated.